### PR TITLE
WIP: admin-guide - chapter 5 - VIM

### PR DESCRIPTION
**Pull request by alemorvan**
## Author checklist (to be completed by original Author)
- [x] Is this document a good fit for the Rocky project ?
- [ ] Is this a non-English contribution? 
- [ ] Title and Author MetaTags have been inserted into the document 
- [x] If applicable, steps and instructions have been tested to work on a real system
- [x] Did you perform an initial self-review to fix basic typos and grammatical correctness

## Rocky Documentation checklist  (to be completed by Rocky team) 
- [x] 1st Pass (Check that document is good fit for project and author checklist completed)
- [x] 2nd Pass (Technical Review - check for technical correctness) 
- [x] 3rd Pass (Basic Editorial Review)
- [x] 4th Pass (Detailed Editorial Review and Peer Review)
- [ ] 5th Pass (Include document in TOC)
- [ ] Final pass/approval (Final Review)

### CODE REVIEW

**Comment by alemorvan** [comment]
@sspencerwire could you review this one ? Thx

File: en/rocky/8/admin_guide/05-vi.md [hunk 1]
```diff
@@ -0,0 +1,479 @@
+# VI Text Editor
+
+In this chapter you will learn how to work with the VIsual editor.
+
+****
+**Objectives** : In this chapter, futurs Linux administors will learn how to:
```
Comment (SergeCroise) [inline en/rocky/8/admin_guide/05-vi.md:6]: ...future Linux administrators ...

File: en/rocky/8/admin_guide/05-vi.md [hunk 2]
```diff
@@ -0,0 +1,479 @@
...
+:checkered_flag: **user commands**, **linux**
+
+**Knowledge**: :star: \
+**Complexity**: :star: :star:\
+**Reading time**: 20 minutes
+****
+
+*Visual* (**VI**) is a very popular text editor under Linux, despite its limited ergonomics. It is indeed an editor entirely in text mode: each action is done with a key on the keyboard or dedicated commands.
+
+Very powerful, it is above all very practical since it is on the whole minimal for basic applications. It is therefore accessible in case of system failure. Its *universality* (it is present on all Linux distributions and under Unix) makes it a *crucial* tool for the administrator.
```
Comment (SergeCroise) [inline en/rocky/8/admin_guide/05-vi.md:20]: ?

File: en/rocky/8/admin_guide/05-vi.md [hunk 3]
```diff
@@ -0,0 +1,479 @@
...
+At startup, VI is in *commands* mode.
+
+> :star: **TIP:**
+A line of text is ended by pressing <kbd>ENTER</kbd> but if the screen is not wide enough, VI makes automatic line breaks, _wrap_ configuration by default. These line breaks may not be desired, this is the _nowrap_ configuration.
+
+To exit VI, from the Commands mode, tap <kbd>:</kbd> then type:
+
+* `q` to exit without saving (_quit_);
+* `w` to save your work (_write_);
+* `wq` (_write quit_) or `x` (_eXit_) to exit and save.
```
Comment (SergeCroise) [inline en/rocky/8/admin_guide/05-vi.md:74]: ... to save and exit.

File: en/rocky/8/admin_guide/05-vi.md [hunk 4]
```diff
@@ -0,0 +1,479 @@
...
+
+* delete,
+* replace,
+* copy,
+* cut,
+* paste.
+
+These operations are done in *command* mode.
+
+### Caractères
```
Comment (SergeCroise) [inline en/rocky/8/admin_guide/05-vi.md:244]: ### Characters

File: en/rocky/8/admin_guide/05-vi.md [hunk 5]
```diff
@@ -0,0 +1,479 @@
...
+
+## EX commands
+
+The *Ex* mode allows you to act on the file (saving, layout, options, ...). It is also in *Ex* mode that search and replace commands are entered. The commands are displayed at the bottom of the page and must be validated with the <kbd>ENTER</kbd> key.
+
+To switch to *Ex* mode, from *commands* mode, type <kbd>:</kbd>.
+
+### Number the lines
+
+* Show/hide dialing:
```
Comment (SergeCroise) [inline en/rocky/8/admin_guide/05-vi.md:353]: ?
numbering

File: en/rocky/8/admin_guide/05-vi.md [hunk 6]
```diff
@@ -0,0 +1,479 @@
...
+
+This is the default mode when VI starts up. To access it from any of the other modes, simply press the <kbd>ESC</kbd> key.
+
+All entries are interpreted as commands and the corresponding actions are executed. These are essentially commands for editing text (copy, paste, undo, ...).
+
+The commands are not displayed on the screen.
+
+### The Insert mode
+
+This is the text modification mode. To access it from the *commands* mode, you have to press special keys that will perform an action in addition to changing the mode.
```
Comment (SergeCroise) [inline en/rocky/8/admin_guide/05-vi.md:103]: *command* mode

File: en/rocky/8/admin_guide/05-vi.md [hunk 7]
```diff
@@ -0,0 +1,479 @@
...
+
+### The Ex mode
+
+This is the file modification mode. To access it, you must first switch to *command* mode, then enter the *ex* command frequently starting with the character `:`.
+
+The command is validated by pressing the <kbd>ENTER</kbd> key.
+
+## Moving the cursor
+
+In *commands* mode, there are several ways to move the cursor.
```
Comment (SergeCroise) [inline en/rocky/8/admin_guide/05-vi.md:115]: *command* mode

File: en/rocky/8/admin_guide/05-vi.md [hunk 8]
```diff
@@ -0,0 +1,479 @@
...
+
+The mouse is not active in a text environment but is in a graphic environment, it is possible to move it character by character, but shortcuts exist to go faster.
+
+VI remains in *commands* mode after moving the cursor.
```
Comment (SergeCroise) [inline en/rocky/8/admin_guide/05-vi.md:119]: *command* mode

**Review by SergeCroise — CHANGES_REQUESTED** [review]

File: en/rocky/8/admin_guide/05-vi.md (hunk 1 above)
Comment (alemorvan) [reply en/rocky/8/admin_guide/05-vi.md:6]:
```suggestion
**Objectives** : In this chapter, future Linux administors will learn how to:
```

File: en/rocky/8/admin_guide/05-vi.md (hunk 3 above)
Comment (alemorvan) [reply en/rocky/8/admin_guide/05-vi.md:74]:
```suggestion
* `wq` (_write quit_) or `x` (_eXit_) to save and exit.
```

File: en/rocky/8/admin_guide/05-vi.md (hunk 4 above)
Comment (alemorvan) [reply en/rocky/8/admin_guide/05-vi.md:244]:
```suggestion
### Characters
```

File: en/rocky/8/admin_guide/05-vi.md (hunk 6 above)
Comment (alemorvan) [reply en/rocky/8/admin_guide/05-vi.md:103]:
```suggestion
This is the text modification mode. To access it from the *command* mode, you have to press special keys that will perform an action in addition to changing the mode.
```

File: en/rocky/8/admin_guide/05-vi.md (hunk 7 above)
Comment (alemorvan) [reply en/rocky/8/admin_guide/05-vi.md:115]:
```suggestion
In *command* mode, there are several ways to move the cursor.
```

Comment (alemorvan) [reply en/rocky/8/admin_guide/05-vi.md:115]: Fix: again "command mode"

File: en/rocky/8/admin_guide/05-vi.md (hunk 8 above)
Comment (alemorvan) [reply en/rocky/8/admin_guide/05-vi.md:119]:
```suggestion
VI remains in *command* mode after moving the cursor.
```

Comment (alemorvan) [reply en/rocky/8/admin_guide/05-vi.md:119]: Fix: again "command mode"

File: en/rocky/8/admin_guide/05-vi.md [hunk 9]
```diff
@@ -0,0 +1,479 @@
...
+
+[`Ctrl+R`]
+
+## EX commands
+
+The *Ex* mode allows you to act on the file (saving, layout, options, ...). It is also in *Ex* mode that search and replace commands are entered. The commands are displayed at the bottom of the page and must be validated with the <kbd>ENTER</kbd> key.
+
+To switch to *Ex* mode, from *commands* mode, type <kbd>:</kbd>.
+
+### Number the lines
```
Comment (alemorvan) [inline en/rocky/8/admin_guide/05-vi.md:351]:
```suggestion
### Numbering the lines
```

File: en/rocky/8/admin_guide/05-vi.md (hunk 5 above)
Comment (alemorvan) [reply en/rocky/8/admin_guide/05-vi.md:353]: Modified. Thx.

File: en/rocky/8/admin_guide/05-vi.md (hunk 2 above)
Comment (alemorvan) [reply en/rocky/8/admin_guide/05-vi.md:20]: What did you mean ?

**Comment by alemorvan** [comment]
Thank you  @SergeCroise for your review 👍

File: en/rocky/8/admin_guide/05-vi.md (hunk 2 above)
Comment (SergeCroise) [reply en/rocky/8/admin_guide/05-vi.md:20]: difficult to understand (for me)

**Review by SergeCroise — CHANGES_REQUESTED** [review]

File: en/rocky/8/admin_guide/05-vi.md (hunk 2 above)
Comment (alemorvan) [reply en/rocky/8/admin_guide/05-vi.md:20]: Let's wait for @sspencerwire what he suggests

File: en/rocky/8/admin_guide/05-vi.md [hunk 10]
```diff
@@ -3,7 +3,7 @@
 In this chapter you will learn how to work with the VIsual editor.
 
 ****
-**Objectives** : In this chapter, futurs Linux administors will learn how to:
+**Objectives** : In this chapter, future Linux administors will learn how to:
```
[the remark this answers is not in the document]
Comment (alemorvan) [reply en/rocky/8/admin_guide/05-vi.md:6]: indeed ^^

[the remark this answers is not in the document]
Comment (alemorvan) [reply en/rocky/8/admin_guide/05-vi.md:6]:
```suggestion
**Objectives** : In this chapter, future Linux administrators will learn how to:
```

File: en/rocky/8/admin_guide/05-vi.md [hunk 11]
```diff
@@ -0,0 +1,479 @@
...
+
+### From the first character of a word
+
+Words are made up of letters or numbers. Punctuation characters and apostrophes separate words.
+
+If the cursor is in the middle of a word [`w`] moves to the next word, [`b`] moves to the beginning of the word.
+
+If the line is finished, VI goes automatically to the next line.
+
+* Move one or `n` words to the right :
```
[the remark this answers is not in the document]
Comment (alemorvan) [reply en/rocky/8/admin_guide/05-vi.md:157]:
```suggestion
* Move one or `n` words to the right:
```

File: en/rocky/8/admin_guide/05-vi.md [hunk 12]
```diff
@@ -0,0 +1,479 @@
...
+
+[`w`] or [`n`][`w`]
+
+* Move one or `n` words to the left :
```
[the remark this answers is not in the document]
Comment (alemorvan) [reply en/rocky/8/admin_guide/05-vi.md:161]:
```suggestion
* Move one or `n` words to the left:
```

File: en/rocky/8/admin_guide/05-vi.md [hunk 13]
```diff
@@ -0,0 +1,479 @@
...
+
+[`M`]
+
+* Move to the last line of the screen:
+
+[`L`]
+
+## Inserting text
+
+In *commands* mode, there are several ways to insert text.
```
[the remark this answers is not in the document]
Comment (alemorvan) [reply en/rocky/8/admin_guide/05-vi.md:189]:
```suggestion
In *command* mode, there are several ways to insert text.
```

File: en/rocky/8/admin_guide/05-vi.md [hunk 14]
```diff
@@ -0,0 +1,479 @@
...
+
+## EX commands
+
+The *Ex* mode allows you to act on the file (saving, layout, options, ...). It is also in *Ex* mode that search and replace commands are entered. The commands are displayed at the bottom of the page and must be validated with the <kbd>ENTER</kbd> key.
+
+To switch to *Ex* mode, from *commands* mode, type <kbd>:</kbd>.
+
+### Numbering the lines
+
+* Show/hide dialing:
```
[the remark this answers is not in the document]
Comment (alemorvan) [reply en/rocky/8/admin_guide/05-vi.md:353]:
```suggestion
* Show/hide numbering:
```

File: en/rocky/8/admin_guide/05-vi.md [hunk 15]
```diff
@@ -0,0 +1,479 @@
...
+
+* Search for a string before the cursor:
+
+`?string`
+
+* Go to the next occurrence found:
+
+[`n`]
+
+* Go to the previous occurrence found :
```
[the remark this answers is not in the document]
Comment (alemorvan) [reply en/rocky/8/admin_guide/05-vi.md:373]:
```suggestion
* Go to the previous occurrence found:
```

File: en/rocky/8/admin_guide/05-vi.md [hunk 16]
```diff
@@ -0,0 +1,479 @@
...
+
+* Cancel a cancellation
+
+[`Ctrl+R`]
+
+## EX commands
+
+The *Ex* mode allows you to act on the file (saving, layout, options, ...). It is also in *Ex* mode that search and replace commands are entered. The commands are displayed at the bottom of the page and must be validated with the <kbd>ENTER</kbd> key.
+
+To switch to *Ex* mode, from *commands* mode, type <kbd>:</kbd>.
```
[the remark this answers is not in the document]
Comment (alemorvan) [reply en/rocky/8/admin_guide/05-vi.md:349]:
```suggestion
To switch to *Ex* mode, from *command* mode, type <kbd>:</kbd>.
```

File: en/rocky/8/admin_guide/05-vi.md [hunk 17]
```diff
@@ -0,0 +1,479 @@
...
+
+[`O`]
+
+* Inserting text after a line:
+
+[`o`]
+
+## Characters, words and lines
+
+VI allows text editing by managing :
```
[the remark this answers is not in the document]
Comment (alemorvan) [reply en/rocky/8/admin_guide/05-vi.md:228]:
```suggestion
VI allows text editing by managing:
```

**Comment by alemorvan** [comment]
All is ok for me

**Pull request merged**